src/peft/tuners/xlora/model.py

```diff
@@ -277,6 +277,8 @@ def __getattr__(self, name: str):
         try:
             return super().__getattr__(name)  # defer to nn.Module's logic
         except AttributeError:
+            if name == "base_model":  # see #1892: prevent infinite recursion if class is not initialized
```

Comment:
In this case, it should be `if name == "lora_model":`. The idea is that if `self.lora_model` is not yet set, we want to avoid infinite recursion.

Reply:
Ah ok! I fixed that.